tools/mentors/page_text_image.js

```diff
@@ -0,0 +1,45 @@
+/* global $ pageBrowserData topLine viewSplitter imageControl textControl pageChanger splitControl
+roundSelect proofIntData projectInput projectSelectButton */
+
+$(function () {
+    let topDiv = $("#top-div");
+    let fixHead = $("<div>", {class: 'fix control-form'});
+    let pageControlForm = $("<form>", {method: "get"});
+    topDiv.append(fixHead);
+
+    if(pageBrowserData.errorMessage) {
+        fixHead.append($("<p>", {class: 'error'}).append(pageBrowserData.errorMessage));
+    }
+    fixHead.append(pageControlForm);
+    if(!pageBrowserData.projectid) {
+        // just show the project input
+        pageControlForm.append($("<p>").append(proofIntData.strings.selectAProject), projectInput(), projectSelectButton());
+    } else if(!pageBrowserData.currentPage) {
+        // show project name, and page selector
+        pageControlForm.append(topLine(), pageChanger(), " ", roundSelect(false));
+    } else {
+        // if a page is given show it
+        let stretchDiv = $("<div>", {class: 'stretch'});
+        topDiv.append(stretchDiv);
+
+        let theImageControl = imageControl();
+        let imageDiv = $("<div>", {class: 'overflow-auto image-back'}).append(theImageControl.image);
+        theImageControl.setZoom();
+
+        let theTextControl = textControl();
+        let topTextDiv = $("<div>").append(theTextControl.textArea);
+        let blankDiv = $("<div>");
+        let textDiv = $("<div>").append(topTextDiv, blankDiv);
+
+        stretchDiv.append(imageDiv, textDiv);
+        let theSplitter = viewSplitter(stretchDiv);
+
+        pageControlForm.append(topLine(), pageChanger(), " ", roundSelect(true), " ", theImageControl.controls,
+            theSplitter.buttons, theTextControl.controls);
+
+        let subSplitter = splitControl();
```

Comment:
What is the bottom half of the right column for? I wasn't able to get anything to ever show up in there and the original doesn't have it.

Reply:
This is so that mentors can restrict the number of lines of text visible which is currently possible in most browsers as requested in this comment: https://www.pgdp.net/phpBB3/viewtopic.php?f=4&t=66584&hilit=page+match&start=15#p1178414

Reply:
I question the utility of this, but let's see how the other mentors like it.